momento-sdk/src/main/java/momento/sdk/ScsDataClient.java

```diff
@@ -388,9 +391,7 @@ CompletableFuture<CacheListConcatenateBackResponse> listConcatenateBackString(
     try {
       checkCacheNameValid(cacheName);
       checkListNameValid(listName);
-      if (ttl == null) {
```

Comment:
Where is this null check being done now?

Reply:
When building the requests, we use:
`.setTtlMilliseconds(ttl.toMilliseconds().orElse(itemDefaultTtl.toMillis()))`

which would mean to use the itemDefaultsTtl value if the tt.toMilliseconds is null, right?